src/lib/features/personal-dashboard/personal-dashboard-read-model-type.ts

```diff
@@ -1,20 +1,24 @@
 import type { ProjectOwners } from '../project/project-owners-read-model.type';
 
 export type PersonalFeature = { name: string; type: string; project: string };
-export type PersonalProject = {
+export type BasePersonalProject = {
```

Comment:
we will remove this type once we finish migration to the single project endpoint

Reply:
are we also removing the optional roles, then?

Reply:
I was thinking that we can read the roles lazily when you open single project together with owner, events and onboarding status fetched from the single project API in personal dashboard controller.
I kept owners and roles optional to allow easier migration (data structure copy paste to single project endpoint)